packages/api/src/index.js

```diff
@@ -27,7 +27,8 @@ const auth = {
   '🤲': handler => withCorsHeaders(handler),
   '🔒': handler => withCorsHeaders(withApiOrMagicToken(handler)),
   '👮': handler => withCorsHeaders(withMagicToken(handler)),
-  '📌': handler => auth['🔒'](withPinningAuthorized(handler))
+  '📌': handler => withPinningAuthorized(handler),
+  '🚫': handler => withAccountNotRestricted(handler)
```

Comment:
Curious if you considered any way of combining these 2 when needed. It would be great if we had a clean way of not needing two HTTP Req + 2 SQL ops just to authorize something. 

We would need to add extra logic to DB Client, probably options to changed functions to allow combining requests with multiple tags at once

Reply:
I had this thought and I think it's a good optimisation to aim for. Do you think this is a blocker for now? I'd rather tackle it in a separate PR/ticket to keep this one from dragging on too much, but let me know if it's better to do it here.

Reply:
Aye, I think just add an issue for that work is all that's needed here.

Reply:
Tracking [here](https://github.com/web3-storage/web3.storage/issues/1162)